serial.c

```diff
@@ -152,10 +156,16 @@ int open_comport()
    }
 #elif TERMIOS
     char tmp[54];
-    if( comport.number < 100 )
-        snprintf(tmp, sizeof(tmp), "/dev/ttyS%d", comport.number);
-    else
-        snprintf(tmp, sizeof(tmp), "/dev/ttyUSB%d", comport.number-100);
+    if (comport.number >= 1000) {
```

Comment:
I'm fine with this addition, but why would someone want this?

Reply:
Oh, I simply use it to test [ELM327-emulator](https://github.com/Ircama/ELM327-emulator#scantool).

Reply:
Ah-ha! That's worth noting in the USAGE.md.